modules/profile/manifests/jenkins/agent_gpu.pp

```diff
@@ -38,8 +38,10 @@
     ensure  => installed,
   }
 
+  $busid = generate('nvidia-xconfig --query-gpu-info  | grep BusID | sed "s/.*PCI:/PCI:/g"')
+
   file { '/etc/X11/xorg.conf':
-    source  => 'puppet:///modules/profile/jenkins/agent_gpu/etc/X11/xorg.conf',
+    content => template('agent_files/xorg.conf.erb'),
```

Comment:
Does this template file actually exist?

Reply:
I think so: https://github.com/ros-infrastructure/buildfarm_deployment/pull/211/files#diff-d055f30477ce9efaa7d5538e0cfe190e